game_eggs/steamcmd_servers/american-truck-simulator/egg-ats.json

```diff
@@ -0,0 +1,31 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2022-12-10T21:09:36-05:00",
+    "name": "American Truck Simulator Dedicated Server.",
+    "author": "chase@cstmgames.com",
+    "description": "This will set up SteamCMD and all the required files for the newly released American Truck Simulator Dedicated server.  Simply placed the exported server config from your ATS instance to the .local\/share\/American Truck Simulator\/ folder and select start.",
+    "features": null,
```

Comment:
add the steam diskspace feature here

Reply:
I am unfamillure with that feature, could you point me in the right documentation direction and I would be glad to see what I can do.

Reply:
ofc, it is something that you have to add manual to the egg when it is exported. It tells the panel when it sees a special message in the console to warn the end user that the server ran out of disk space. you can not see it in the panel . Have a look here: <https://github.com/parkervcp/eggs/blob/master/game_eggs/steamcmd_servers/open_fortress/egg-open-fortress.json> L 11-13

Reply:
Ah!  Make perfect sense.  I will get those added!